### PR TITLE
day 1 work pushed

### PR DESCRIPTION
**Pull request by keerthanaraghu**
day 1 assignment - pull request

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.assignment.ctomatrix</groupId>
+  <artifactId>ctomatrix</artifactId>
+  <version>0.0.1-SNAPSHOT</version>
+  
+  
+  <dependencies>
```
Comment (raghukraman) [inline pom.xml:8]: Add logger dependency , driver dependency(may be later) and format the file

File: src/main/java/com/assignment/entities/FeedBySKU.java [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+public class FeedBySKU {
+	
+	private String sku;
+	private Integer optionGroupID;
+	private String optionID;
+	private String salesOrg;
+	private String mpn;
+	private String description;
+	private String publishedDescr;
+	private String dispyOnWeb;
```
Comment (raghukraman) [inline src/main/java/com/assignment/entities/FeedBySKU.java:19]: correct the spelling.

File: src/main/java/com/assignment/entities/FeedBySKU.java [hunk 3]
```diff
@@ -0,0 +1,85 @@
+package com.assignment.entities;
+
+import java.util.Date;
+
+public class FeedBySKU {
```
Comment (raghukraman) [inline src/main/java/com/assignment/entities/FeedBySKU.java:10]: Add java doc
This class represents the <table name>

File: src/main/java/com/assignment/xml/vo/CTOMatrix.java [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+public class CTOMatrix {
+	
+	@XmlElement (name = "KMATS")
+	private List<KMat> kmats;
+	
+	@XmlElement (name = "FIND_NOS")
+	private List<FindNos> findNos;
+	
+	@XmlElement (name = "Options")
```
Comment (raghukraman) [inline src/main/java/com/assignment/xml/vo/CTOMatrix.java]: This is not matching with the xml. please check

File: src/main/java/com/assignment/xml/vo/CTOMatrix.java [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+	private Option options;
+	
+	@XmlElement (name = "SALES_ORGS")
```
Comment (raghukraman) [inline src/main/java/com/assignment/xml/vo/CTOMatrix.java]: @keerthanaraghu not matching with the xml structure. please check

**Review by raghukraman — COMMENTED** [review]
@keerthanaraghu Please incorporate review comments

**Comment by raghukraman** [comment]
@keerthanaraghu , next time create some meaning branch names like "create-vo", "create-vo-entity", "cto-matrix-mapper-feature", "save-to-db-feature" etc and not like "keerthana"

File: src/main/java/com/assignment/entities/FeedBySKU.java (hunk 3 above)
Comment (keerthanaraghu) [reply src/main/java/com/assignment/entities/FeedBySKU.java:10]: done

**Review by raghukraman — APPROVED** [review]

**Pull request merged**